src/storage/mongodb/UserStorage.ts

```diff
@@ -845,7 +847,9 @@ export default class UserStorage {
         tenantID, aggregation: aggregation, localField: '_id', foreignField: 'tagID',
         count: true, asField: 'transactionsCount', oneToOneCardinality: false,
         objectIDFields: ['createdBy', 'lastChangedBy']
-      });
+      }, [{
+        '$match': { 'userID': { $exists: true, $ne: null } }
+      }]);
```

Comment:
Nbr of transaction is related to tags then this filter is not right.
Use the param withUser flag to set this one.